magneticow/magneticow/templates/homepage.html

```diff
@@ -15,5 +15,9 @@
         <input type="search" name="search" placeholder="Search the BitTorrent DHT" autofocus>
     </form>
 </main>
+
+<footer>
+  <div>Now with {{counter[0][0]}} torrents available !</div>
```

Comment:
See the previous comment above. This line should ideally become:

    <div>Now with {{n_torrents}} torrents available !</div>

or whatever name you think is the best fit.